packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -294,7 +309,34 @@ export async function readSchema(projectDirectory: string): Promise<{schema: str
     schema,
     modelToDatasourceMap,
   };
-}
+};
+
+const removeAmplifyInput = (schema: string): SchemaReaderConfig => {
```

Comment:
Don't we already have this utility somewhere? Did I miss it getting moved from somewhere else.

Reply:
I couldn't find it. For the 'AMPLIFY' input object in schema.graphql file, we skip if while processing the AST. (Code reference link below).
https://github.com/aws-amplify/amplify-category-api/blob/9a4f6203095d87ce553e4cc6a4c9210fcc5b2fe0/packages/amplify-graphql-transformer-core/src/transformer-context/output.ts#L77